server/server.js

```diff
@@ -0,0 +1,23 @@
+const { createServer } = require("https");
+const { parse } = require("url");
+const next = require("next");
+const fs = require("fs");
+const path = require("path");
+const dev = process.env.NODE_ENV !== "production";
+const app = next({ dev });
+const handle = app.getRequestHandler();
+
+const httpsOptions = {
+  key: fs.readFileSync(path.join(__dirname, './localhost+1-key.pem')),
+  cert: fs.readFileSync(path.join(__dirname, './localhost+1.pem')),
+};
+
+app.prepare().then(() => {
+  createServer(httpsOptions, (req, res) => {
+    const parsedUrl = parse(req.url, true);
+    handle(req, res, parsedUrl);
+  }).listen(3000, (err) => {
+    if (err) throw err;
+    console.log("> Server started on https://localhost:3000");
+  });
+});
```

Comment:
it's generally good practice to insert a newline at the end of each file.

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

editorconfig is a great tool that can auto-insert newlines on save. it can also configure other annoyances like tabbing defaults for different file types. https://editorconfig.org/

Reply:
I think I was able to correct this by editing the .gitconfig file in my system. Did so by changing core.autocrlf from input to true (on Widnows).